engine/src/main/java/org/destinationsol/common/SolDescriptiveException.java

```diff
@@ -5,22 +5,21 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Any reason for this indentation?

Reply:
Screw you IDEA

Reply:
Actually, tried to make a temp class and the indentation was already inserted, so no misconfigured autoreformat. Maybe some mistake in template?